src/MultiRegion/multi_region_cubed_sphere_grid.jl

```diff
@@ -226,7 +227,7 @@ function ConformalCubedSphereGrid(arch::AbstractArchitecture=CPU(), FT=Float64;
             end
 
             if $(horizontal_topology) == FullyConnected
-                fill_cubed_sphere_halo_regions!($(Symbol(field)), (Center(), Center()))
+                fill_cubed_sphere_halo_regions!($(Symbol(field)))
```

Comment:
hm... you define a `fill_cubed_sphere_halo_regions!() = nothing` and then you call it in the same file. So it's doing exactly nothing, right?